.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 
 language: rust
 cache: cargo
+env:
+  - RUSTFLAGS="-C incremental=false"
```

Comment:
wdyt about passing `-C debuginfo=0` as well? It seems unlikely this is at all useful to us in CI; and it reduces the artifact size from ~2G to ~30M.

Perhaps incremental can be enabled if we turn off debug info?

Reply:
On my Linux laptop, from a clean build:

```
:; make test >/dev/null 2>&1 && du -sh target
6.6G    target
:; env RUSTFLAGS="-C debuginfo=0" make test >/dev/null 2>&1 && du -sh target
1.2G    target
```

Reply:
Ah, I thought debuginfo had already been disabled in CI, and I was still destroying it...